### PR TITLE
Default directory for contrib modules should be sites/all/modules/con…

### PR DESCRIPTION
**Pull request by jbickar**
…trib. Other minor cleanup

### CODE REVIEW

File: make/production/jumpstart-academic.make [hunk 1]
```diff
@@ -1,5 +1,7 @@
 core = 7.x
 api = 2
+; Default directory should be sites/all/modules/contrib
+defaults[projects][subdir] = contrib
```
Comment (sherakama) [inline make/production/jumpstart-academic.make:4]: I can't remember why at this point in time but I had a reason to not do this in the Deployer make files. Let me see if I can shake out the cobwebs.

File: make/production/jumpstart-personal.make [hunk 2]
```diff
@@ -1,5 +1,7 @@
 core = 7.x
 api = 2
+; Default directory should be sites/all/modules/contrib
+defaults[projects][subdir] = contrib
```
Comment (sherakama) [inline make/production/jumpstart-personal.make:4]: This project has fallen so far behind it is not worth supporting any further. I would avoid updates to it in the future.

Comment (jbickar) [inline make/production/jumpstart-personal.make:4]: @jbickar does not like this

File: make/production/jumpstart-academic.make (hunk 1 above)
Comment (sherakama) [inline make/production/jumpstart-academic.make:4]: It may have been for the 'stanford' stuff but that is sorted now.

**Pull request merged**